### ISSUE TITLE
Unnecessary show conflicting fields checkbox in merging subview dialogs 

### ISSUE THREAD

**Issue by bronwyncombs** [specify/specify7#3879]
**Describe the bug**
A "Show conflicting fields only" checkbox appears on the record-editing sub-dialogs from large buttons in Merge Records dialog. It is for the main Merge Records dialog and shouldn't be on these also.

**Screenshots**

https://github.com/specify/specify7/assets/135047322/8e556318-e27c-4245-958e-0f905ac12da8

**Coming from a button for a field that was not conflicting, it exits the dialog:**

https://github.com/specify/specify7/assets/135047322/a4a14a80-e074-4e54-94d5-1ff562a24de8

**Desktop:**
 - macOS, Chrome, `v7.9-dev`

**Database Name:** `archireef`

**Comment by maxxxxxdlp** [comment]
hmm, the fact that the checkbox is there is not a bug
hmm, the fact that the checkbox has no effect is the bug

**Comment by grantfitzsimmons** [comment]
I think @maxxxxxdlp is right on this one.

**Comment by CarolineDenis** [comment]
Why authorizing to open the SubView when the count is 0?

**Comment by CarolineDenis** [comment]
> Why authorizing to open the SubView when the count of records is 0?

@melton-jason @grantfitzsimmons

**Comment by maxxxxxdlp** [comment]
because when you open the subview, you can add records to it

**Comment by CarolineDenis** [comment]
When you click on "Add" in the merge dialog, a form will open with all fields. However, note that the "Show conflicting fields only" checkbox does not have any effect on this new form.

In my opinion, the checkbox should not be displayed when there are zero records in the merge dialog.

**Comment by maxxxxxdlp** [comment]
@CarolineDenis yes good catch! now I remember this - if there are 0 or 1 records, then that checkbox has no effect as "show conflicting fields only" only makes sense when where are 2+ records
